app/components/post_list/post/body/acknowledgements/index.tsx

```diff
@@ -0,0 +1,183 @@
+    location: string;
+    post: PostModel;
+    theme: Theme;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            alignItems: 'center',
+            borderRadius: 4,
+            backgroundColor: changeOpacity(theme.onlineIndicator, 0.12),
+            flexDirection: 'row',
+            height: 32,
+            justifyContent: 'center',
+            paddingHorizontal: 8,
+        },
+        containerActive: {
+            backgroundColor: theme.onlineIndicator,
+        },
+        text: {
+            ...typography('Body', 100, 'SemiBold'),
+            color: theme.onlineIndicator,
+        },
+        textActive: {
+            color: '#fff',
+        },
+        icon: {
+            marginRight: 4,
+        },
+        divider: {
+            width: 1,
+            height: 32,
+            marginHorizontal: 8,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.16),
+        },
+        listHeader: {
+            marginBottom: 12,
+        },
+        listHeaderText: {
+            color: theme.centerChannelColor,
+            ...typography('Heading', 600, 'SemiBold'),
+        },
+    };
+});
+
+const Acknowledgements = ({currentUser, hasReactions, location, post, theme}: Props) => {
+    const intl = useIntl();
+    const isTablet = useIsTablet();
+    const {bottom} = useSafeAreaInsets();
+    const serverUrl = useServerUrl();
+
+    const style = getStyleSheet(theme);
+
+    const isCurrentAuthor = post.userId === currentUser.id;
+    const acknowledgements = post.metadata?.acknowledgements || [];
+
+    let acknowledgedAt = 0;
+    if (acknowledgements.length) {
+        const ack = acknowledgements.find((item) => item.user_id === currentUser.id);
+        if (ack) {
+            acknowledgedAt = ack.acknowledged_at;
+        }
+    }
+
+    const handleOnPress = useCallback(() => {
+        if ((acknowledgedAt && moreThan5minAgo(acknowledgedAt)) || isCurrentAuthor) {
+            return;
+        }
+        if (acknowledgedAt) {
+            unacknowledgePost(serverUrl, post.id);
+        } else {
+            acknowledgePost(serverUrl, post.id);
+        }
+    }, [acknowledgedAt, isCurrentAuthor, post, serverUrl]);
+
+    const handleOnLongPress = useCallback(() => {
+        if (!acknowledgements.length) {
+            return;
+        }
+        const userAcknowledgements: Record<string, number> = {};
+        const userIds: string[] = [];
+
+        acknowledgements.forEach((item) => {
+            userAcknowledgements[item.user_id] = item.acknowledged_at;
+            userIds.push(item.user_id);
+        });
+
+        const renderContent = () => (
+            <>
+                {!isTablet && (
+                    <View style={style.listHeader}>
+                        <FormattedText
+                            id='mobile.participants.header'
+                            defaultMessage={'Thread Participants'}
```

Comment:
title is wrong, it should be `Acknowledgements` based on the design
![image](https://user-images.githubusercontent.com/6757047/222080014-ee851288-87f3-4ee5-952d-4ef6e2d1a6eb.png)